packages/plugin-zod/src/parser/index.ts

```diff
@@ -171,7 +171,7 @@ type ParserOptions = {
   coercion?: boolean | { dates?: boolean; strings?: boolean; numbers?: boolean }
 }
 
-export function parse(parent: Schema | undefined, current: Schema, options: ParserOptions): string | undefined {
+export function parse(parent: Schema | undefined, current: Schema, options: ParserOptions, siblings: Schema[]): string | undefined {
```

Comment:
we can probably refactor this as follow:
(not sure about the name `ParserAst`)

```
type ParserAst=  {
   parent: Schema | undefined,
   current: Schema
   siblings: Schema[]
}
export function parse({ current, parent, siblings } : ParserAst , options: ParserOptions): string | undefined {
```

Reply:
I've given it a go, in the newest version here